src/main/java/io/inisos/bank4j/validator/constraintvalidators/Iso20022CharacterSetValidator.java

```diff
@@ -0,0 +1,15 @@
+package io.inisos.bank4j.validator.constraintvalidators;
+
+import io.inisos.bank4j.util.Iso20022ReferenceElementValidator;
+import io.inisos.bank4j.validator.constraints.Iso20022CharacterSet;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class Iso20022CharacterSetValidator implements ConstraintValidator<Iso20022CharacterSet, String> {
+    
+    @Override
+    public boolean isValid(String value, ConstraintValidatorContext constraintValidatorContext) {
+        return Iso20022ReferenceElementValidator.isValidCharacterSet(value);
```

Comment:
validation should pass if value is null (allowing to annotate nullable fields)
but it's ok to fail if value is empty.
adding the following before return in this method LGTM:
```java
if(value == null)
    return true;
return Iso20022ReferenceElementValidator.isValidCharacterSet(value);
```